rocketpy/Flight.py

```diff
@@ -52,6 +52,15 @@ class Flight:
         Flight.timeIterator : function
             Helper iterator function to generate time discretization points.
 
+        Helper parameters:
```

Comment:
Yes, I've assumed the nozzle to be aligned with the beginning of the rail. I believe this is more practical for members from Projeto Jupiter, since our last three rockets had that behavior. I will better document this condition.